Content.Server/CriminalRecords/Systems/CriminalRecordsConsoleSystem.cs

```diff
@@ -229,4 +246,31 @@ private string RecordName(StationRecordKey key)
 
         return record.Name;
     }
+
+    /// <summary>
+    /// Checks if the new identity's name has a criminal record attached to it, and gives the entity the icon that
+    /// belongs to the status if it does.
+    /// </summary>
+    public void CheckNewIdentity(Entity<IdentityComponent> ent)
+    {
+        var name = Identity.Name(ent, _entityManager);
+        var stations = _station.GetStations();
+
+        foreach (var station in stations)
```

Comment:
check the owning station, it doesnt make sense that someone from a different station gets a hud icon for a record this station isnt able to access or edit

Reply:
owning stations doesn't work if the entity you want to see the icon of is not on the grid

Reply:
store it on the sechud and only change station if setting foot on a different one

but i think that might be exactly what StationMember is idk

Reply:
GetStationInMap worked, but it will break if we ever get two stations in one map